src/layouts/Upgrade.tsx

```diff
@@ -1,3 +1,417 @@
+      top={{ lg: 24 }}
+      left={{ lg: 24 }}
+      zIndex={10}
+      {...props}
+    />
+  )
+}
+
+// Note: you must pass components to MDXProvider in order to render them in markdown files
+// https://www.gatsbyjs.com/plugins/gatsby-plugin-mdx/#mdxprovider
+export const upgradeComponents = {
+  a: MdLink,
+  h1: Heading1,
+  h2: Heading2,
+  h3: Heading3,
+  h4: Heading4,
+  p: Paragraph,
+  pre: Pre,
+  Badge,
+  // BeaconChainActions,
+  ButtonLink,
+  Card,
+  Divider,
+  Emoji,
+  ExpandableCard,
+  GlossaryTooltip,
+  InfoBanner,
+  MergeArticleList,
+  MergeInfographic,
+  QuizWidget,
+  RandomAppList,
+  ShardChainsList,
+  UpgradeStatus,
+  YouTube,
+  ...mdxTableComponents,
+  // Contributors,
+  // Logo,
+  // MeetupList,
+}
+
+interface IProps extends ChildOnlyProp, PageContent {
+  frontmatter: UpgradeFrontmatter
+}
+export const UpgradeLayout: React.FC<IProps> = ({
+  children,
+  frontmatter,
+  slug,
+  tocItems,
+  lastUpdatedDate
+}) => {
+  // TODO: Re-enabled after i18n is implemented
+  // const { t } = useTranslation()
+  // const { language } = useI18next()
+  const language = "en"
+
+  const isRightToLeft = isLangRightToLeft(frontmatter.lang as Lang)
+
+  const summaryPoints = getSummaryPoints(frontmatter)
+
+  const dropdownLinks: ButtonDropdownList = {
+    text: "Guide to Ethereum upgrades", // t("page-upgrades-upgrades-guide"),
+    ariaLabel: "Ethereum upgrades menu", // t("page-upgrades-upgrades-aria-label"),
+    items: [
+      {
+        text: "The Beacon Chain", // t("page-upgrades-upgrades-beacon-chain"),
+        to: "/roadmap/beacon-chain/",
+        matomo: {
+          eventCategory: "upgrade menu",
+          eventAction: "click",
+          eventName: "/roadmap/beacon-chain/",
+        },
+      },
+      {
+        text: "The Merge", // t("page-upgrades-upgrades-docking"),
+        to: "/roadmap/merge/",
+        matomo: {
+          eventCategory: "upgrade menu",
+          eventAction: "click",
+          eventName: "/roadmap/merge/",
+        },
+      },
+    ],
+  }
+
+  const lgBreakpoint = useToken("breakpoints", "lg")
+
+  return (
+    <Container>
+      <HeroContainer>
+        <TitleCard>
+          <Breadcrumbs slug={slug} startDepth={1} mt={2} mb="8" />
+          <Title>{frontmatter.title}</Title>
+          <Box>
+            <List listStyleType="disc">
+              {summaryPoints.map((point, idx) => (
+                <SummaryPoint key={idx}>{point}</SummaryPoint>
+              ))}
+            </List>
+          </Box>
+          <LastUpdated>
+            {/* TODO: Re-enable after i18n implemented */}
+            {/* <Translation id="page-last-updated" />:{" "} */}
+            Page last updated:{" "}
+            {getLocaleTimestamp(language as Lang, lastUpdatedDate!)}
+          </LastUpdated>
+        </TitleCard>
+        {frontmatter.image && (
+          <Image
+            flex="1 1 100%"
+            maxW="min(100%, 816px)"
+            sx={{ objectFit: "cover" }}
```

Comment:
replace this by using [the `style` prop](https://nextjs.org/docs/pages/api-reference/components/image#style)